css/components/grid.css

```diff
@@ -0,0 +1,37 @@
+.grid-50-50-layout {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  border: 1px solid rgb(0, 0, 0, 0.1);
+  gap: 1rem;
+  padding: 1rem;
+  max-width: 50rem;
+  height: auto;
+}
+
+.grid-cont-1 img {
+  width: 100%;
+  height: 100%;
+}
+
+.grid-cont-2 {
+  border: 1px solid rgb(0, 0, 0, 0.2);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.grid-3-column-layout {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  border: 1px solid rgb(0, 0, 0, 0.1);
+  padding: 1rem;
+  gap: 1rem;
+  width: 100%;
+  max-width: 56.5rem;
+  margin: 0 auto;
```

Comment:
You can add justify-items : center for better alignment.

Reply:
have added a separate class for that